srcpkgs/python3-telegram/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'python3-telegram'
+pkgname=python3-telegram
+version=0.14.0
+revision=1
+wrksrc=python-telegram-${version}
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="libtd"
+short_desc="Python library for building Telegram clients"
+maintainer="Maxim Karasev <begs@disroot.org>"
+license="MIT"
+homepage="https://github.com/alexander-akhmetov/python-telegram"
+distfiles="${PYPI_SITE}/p/python-telegram/python-telegram-${version}.tar.gz"
```

Comment:
if it's feasable, please use source tarball from github and don't vendor the license here.